components/promise.rst

```diff
@@ -27,6 +31,9 @@ request with HTTPlug::
 
 See :ref:`message-factory` on how to use message factories.
 
+Although the promise itself is not restricted to resolve a specific result type,
+in HTTP context it resolves a PSR-7 Response or fails with an ``Http\Client\Exception``.
```

Comment:
i would double backtick Response (probably with FQN to be consistent with the exception class)